src/atoms/shortenText/types.ts

```diff
@@ -0,0 +1,10 @@
+import { BodySizes, HeaderSizes } from '../texts/types';
+
+export interface ShortenTextProps {
+  text: string,
+  size: BodySizes | HeaderSizes,
```

Comment:
I just checked and in the frontend repo we use 14-Regular something around 174 times, since this is our most common used font i think you can make size optional and by default fallback to 14-Regular.

Lmk what you think, also @goshatravin

Reply:
yeah I agree with this, maybe we should even go as far as doing this for our <Text /> component too